### DISCUSSION TITLE
ERROR! - Character conversion between CCSID 1200 and CCSID 65535 not valid.  (57017)

### DISCUSSION THREAD

**Discussion by rmarsh221** [codefori/vscode-ibmi#839] [IBM i] [answered]
Hello and thank you in advance. First, I hope I'm in the right place. I installed the IBM i Development Pack from Halcyon-Tech and the source code editor works great. No trouble at all connecting to the IBM i. The Db2 for i also works up to a point. I can add a schema but when I try to list the tables I get "Character conversion between CCSID 1200 and CCSID 65535 not valid.  (57017)" I have been unable to find where I can change the CCSID or what it should be changed to.

Any help is much appreciated.

**Comment by Wright4i** [comment, marked as answer]
You may want to start here: [tips/ccsid.md](https://github.com/halcyon-tech/vscode-ibmi/blob/7d10055b658310f6c489626a7f885a13521b6010/docs/pages/tips/ccsid.md)

**Reply by rmarsh221** [reply]
Thanks. That did the trick. I wasn't sure if changing it at the user profile level would be necessary. 

So am I correct in assuming it's the job on the system started by the Code for IBM i connection? 
I went looking for that job in WRKUSRJOB but it didn't show up. I figured it was based on the connection but was also looking in the extension so see if the CCSID could be specified.

**Reply by Wright4i** [reply]
Awesome, glad it worked! 

Code for IBM i connects via SSH to PASE and executes the commands from there under your user profile. Your user profile change worked but it can also be done at a system level or you can set PASE a env variable. Your user profile was likely set to the system value by default and that should probably change if you don't want to do CHGUSRPRF for each developer.

**Reply by rmarsh221** [reply]
Thanks for the info. I'm a consultant and work remotely so I don't know what the other developers are doing. But for me, VSCode is much easier on my old eyes. Now if I could just get an extension that would show where something is defined like in Visual Studio. Now that would be something.
